QSPIFBlockDevice.cpp

```diff
@@ -441,6 +491,68 @@ bd_size_t QSPIFBlockDevice::size() const
     return _device_size_bytes;
 }
 
+/********************************/
+/*   Different Device Csel Mgmt */
+/********************************/
+static PinName *generate_initialized_active_qspif_csel_arr()
```

Comment:
If max number of devices is static, we can have a static array (initialized with NCs) without the need for this function.

Reply:
NC value is not 0. so I can't initialize the array with {0} , only with a long list of {NC,NC,..}
This way I can just use a loop as long as the MACRO defined to initialize